be/src/storage/rowset/segment.cpp

```diff
@@ -86,10 +86,19 @@ Status Segment::parse_segment_footer(fs::ReadableBlock* rblock, SegmentFooterPB*
     size_t hint_size = footer_length_hint ? *footer_length_hint : 4096;
     size_t footer_read_size = std::min<size_t>(hint_size, file_size);
 
+    if (partial_rowset_footer != nullptr) {
```

Comment:
When will the partial_rowset_footer be a null pointer?

Reply:
> When will the partial_rowset_footer be a null pointer?

In fact,  the partial_rowset_footer is only not null in the  `rewrite` of partial update

Reply:
> When will the partial_rowset_footer be a null pointer?

if the rowset is full rowset which means `txn_meta` is not exist in `rowset_meta`, partial_rowset_footer will be set to nullptr

Reply:
OK